cinch/site.yml

```diff
@@ -31,6 +31,17 @@
   roles:
     - repositories
 
+- name: enable additional package streams on RHEL 8
+  become: true
+  hosts: repositories
+  tasks:
+    - name: enable libselinux-python stream
+      shell: dnf -y module enable libselinux-python
```

Comment:
We  have it install by [jenkins-common role](cinch/roles/jenkins_common/tasks/main.yml) and listed in default [jenkins_common_packages](cinch/roles/jenkins_common/vars/main.yml)

Reply:
How about supporting systems with python3 where the needed(?) package is 'python3-libselinux'   instead of persisting with python 2?